sort.h

```diff
@@ -108,7 +117,7 @@ static int clzll(uint64_t x) {
 static __inline int compute_minrun(const uint64_t size) {
   const int top_bit = 64 - CLZ(size);
   const int shift = MAX(top_bit, 6) - 6;
-  const int minrun = size >> shift;
+  const int minrun = (int)(size >> shift);
```

Comment:
Are all of these casts necessary when are assigning to an `int`?

Also, could you run `make format`? I think the formatting might be off.

Reply:
Not necessary but some compiler like VS give warnings. I have another question, c89 already supported uint64_t and stdint.h?